templates/navigationTitle.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export default function NavigationTitle(props) {
+  const {
+    title
+  } = props;
+
+  return (
+
+    <div className='navigation-title__inner'>
+      {title}
```

Comment:
Should we allow for escaped html characters here? Thinking there may be a potential need to pronounce characters individually through a screen reader.

Reply:
Yes might be best to use `dangerouslySetInnerHTML` (https://reactjs.org/docs/dom-elements.html)